음양 더하기.js

```diff
@@ -0,0 +1,15 @@
+function solution(absolutes, signs) {
+    let answer = 0;
+    
+    for(let i=0; i <= absolutes.length; i++){
+        if(signs[i] ===true){
```

Comment:
```suggestion
        if(signs[i]){
```
signs[i] 가 true 라는 boolean 타입이라면 굳이 비교 연산자를 사용하지 않아도 동작합니다.